internal/feature/feature_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package feature
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestEnable(t *testing.T) {
+	var cool Flag = -1
+
+	t.Run("EnableOnZeroValue", func(t *testing.T) {
+		f := &Flags{}
+		f.Enable(cool)
+
+		want := true
+		got := f.Enabled(cool)
+
+		if diff := cmp.Diff(want, got); diff != "" {
+			t.Errorf("f.Enabled(...): -want, +got:\n%s", diff)
+		}
+	})
+
+	t.Run("EnabledOnZeroValue", func(t *testing.T) {
```

Comment:
Major Nitpick: aren't we checking here that a flag is _not_ enabled on the `Flags` zero value?